### PR TITLE
Remove unused styles from styles.scss

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- I went through each style in styles.scss and search the code base for any instance where it was used. If I couldn't find any, I deleted it and any associated reference.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Search the codebase for each style that was removed.

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4876

### CODE REVIEW

**Comment by petesfrench** [comment]
Do not merge before the vanilla issues have all been merged

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Mostly looks good, one comment in the code

File: static/sass/_pattern_lists.scss
Comment (petesfrench) [reply static/sass/_pattern_lists.scss:212]: Yes, good catch!

**Review by anthonydillon — APPROVED** [review]
Yep, all looks good to me :+1:

**Pull request merged**
